src/test/java/core/basesyntax/service/impl/ParserTest.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+import java.util.function.Function;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ParserTest {
+    private static Function<List<String>, List<FruitTransaction>> parser;
+
+    @BeforeClass
+    public static void setUpFirst() {
```

Comment:
```suggestion
    public static void setUp() {
```
be consistent with setup methods naming all across your solution